tracer/src/Datadog.Trace/Debugger/ProbeLocationType.cs

```diff
@@ -9,6 +9,7 @@ internal enum ProbeLocationType
     {
         Line,
         Method,
+        MethodSpan,
```

Comment:
This seems like it's confusing two things - the method location, and the method type, which means we have a bit of a leaky abstraction. Any chance we could remove this?

Reply:
Done in ebb2b638668a8ffee58604d13ae658037d77b7d8.